docs/skopeo-standalone-sign.1.md

```diff
@@ -4,11 +4,10 @@
 skopeo\-standalone-sign - Debugging tool - Publish and sign an image in one step.
 
 ## SYNOPSIS
-**skopeo standalone-sign** [*options*] _manifest_ _docker-reference_ _key-fingerprint_
+**skopeo standalone-sign** [*options*] _manifest_ _docker-reference_ _key-fingerprint_ **--output**|**-o** _signature_
```

Comment:
The --output option is handled by [*option*].

We don't display options on the SYNOPSIS line.

Reply:
This one is mandatory, so the square-bracketed [option] is not appropriate.

Reply:
… and already in the repo per #1351, not really a part of this PR.

Reply:
😕 I don't think I changed that element .... looking back at a [prior commit](https://github.com/containers/skopeo/commit/22908fb3e8a17c42675a7471c9b365c507a79a3c#diff-eb57f4b9e43694badc7d10fdfa02d65094b136948eb433f003e9c168b0db0833) it looks like that was changed back in late June @rhatdan